File: server-config-examples/eviction/src/main/java/examples/springdata/geode/server/eviction/service/ProductService.java
package examples.springdata.geode.server.eviction.service;

import org.apache.geode.cache.Region;
import org.springframework.beans.factory.annotation.Qualifier;
import org.springframework.stereotype.Component;

@Component
public class ProductService {

	private final Region productRegion;

    public ProductService(@Qualifier("Products") Region productRegion)
	{
		this.productRegion = productRegion;
	}

	public int size(){
    	return productRegion.size();
	}
}

File: server-config-examples/eviction/src/main/kotlin/example/springdata/geode/server/eviction/kt/service/ProductService.kt
package example.springdata.geode.server.eviction.kt.service

import org.apache.geode.cache.Region
import org.springframework.beans.factory.annotation.Qualifier
import org.springframework.stereotype.Component

@Component
class ProductServiceKT(@param:Qualifier("Products") private val productRegion: Region<*, *>) {

    fun size(): Int {
        return productRegion.size
    }
}